internal/sidekick/internal/parser/discovery/methods.go

```diff
@@ -46,6 +47,27 @@ func makeMethod(model *api.API, serviceID string, input *method) (*api.Method, e
 	if err != nil {
 		return nil, err
 	}
+	var uriTemplate string
+	if strings.HasSuffix(doc.ServicePath, "/") {
+		uriTemplate = fmt.Sprintf("%s%s", doc.ServicePath, input.Path)
+	} else {
+		uriTemplate = fmt.Sprintf("%s/%s", doc.ServicePath, input.Path)
+	}
+	uriTemplate = strings.TrimPrefix(uriTemplate, "/")
+	path, err := ParseUriTemplate(uriTemplate)
+	if err != nil {
+		return nil, err
+	}
+	binding := &api.PathBinding{
+		Verb:            input.HTTPMethod,
+		PathTemplate:    path,
+		QueryParameters: map[string]bool{},
+	}
+	for _, p := range input.Parameters {
+		if p.Location != "path" {
```

Comment:
if all the parameters belong in either the path or as query params, maybe we don't need a body? Not sure.